packages/notification-service/src/firebase.ts

```diff
@@ -217,6 +220,9 @@ export function setLastBlockNotified(newBlock: number): Promise<void> | undefine
     return
   }
 
+  // Set the metric tracking this difference
+  metrics.setBlockDelay(newBlock - lastBlockNotified)
```

Comment:
This is only going to compare the latest known block against the latest known block the previous poll interval. It will lose outliers in block delay for specific blocks. 

I think a good way to catch these outliers would be to:
- In `src/blockscout/transfers` move the `setLastBlockNotified` call on line ~162 to anywhere before the call to `notifyForNewTransfers` on line ~158. This updates the `lastBlockNotified` so that we have the value ready when sending notifications.
- In `src/firebase` on the function `sendPaymentNotification` (line ~260) there's already a log that logs the block delay (`lastBlockNotified - blockNumber`). Send that delay to the metrics.

What do you think?